guides/release/testing/testing-application.md

```diff
@@ -117,9 +117,9 @@ Before making an assertion, wait for the execution of each asynchronous helper t
 ```javascript {data-filename=tests/acceptance/new-post-appears-first-test.js}
   test('should add new post', async function(assert) {
     await visit('/posts/new');
-    await fillIn('input.title', 'My new post');
-    await click('button.submit');
-    assert.equal(this.element.querySelector('ul.posts li').textContent, 'My new post');
+    await fillIn('[data-test-post-input-title]', 'My new post');
+    await click('[data-test-post-save]');
```

Comment:
**suggestion (non-blocking)**

In case you want to commit the suggestion above.

```suggestion
    await fillIn('[data-test-field="Title"]', 'My new post');
    await click('[data-test-button="Save"]');
```